content/en/blog/2025/expose-otel-collector-gateway-api/index.md

```diff
@@ -0,0 +1,445 @@
+---
+title: Securely Exposing Your  OTEL Collector in Kubernetes with Gateway API and mTLS
+date: 2025-04-6
+author: >
+  [Vipin Vijaykumar](https://github.com/vipinvkmenon)
+sig: OpenTelemetry JS
+cSpell:ignore: Danielson
+---
+
+# Securely Exposing Your  OTEL Collector in Kubernetes with Gateway API and mTLS
+**Goal:** 
+
+Exposing an OpenTelemetry (OTEL) Collector running inside Kubernetes to the outside world securely, using the Kubernetes Gateway API and mutual TLS (mTLS) for authentication and encryption. 
+
+As observability becomes increasingly critical in modern distributed systems, centralizing telemetry data via an OTEL Collector is common practice. Often, services or agents running *outside* your Kubernetes cluster need to send data *to* this collector. Exposing internal services requires careful consideration of security and standardization. This is where the Kubernetes Gateway API and mTLS shine.
+
+Typicall this kind of a setup is useful when you have applications/ workloads that are external to the cluster and you need to collect the telemetry data. Some examples:
```

Comment:
```suggestion
Typically this kind of a setup is useful when you have applications/ workloads that are external to the cluster and you need to collect the telemetry data. Some examples:
```

Our spell checker provides you with some suggestions on wording issues, please review them and fix them accordingly (e.g. also the repeated issue of "OTEL" vs "OTel"